### ISSUE TITLE
How to disable cookies in signalmaster to solve Chrome Error: cookies with cross-site requests

### ISSUE THREAD

**Issue by KevCoLaboratories** [simplewebrtc/signalmaster#141]
I've had my Signal-Master server up and running on an EC2 server running node.js for a couple years now, but recently I've been receiving a warning in Chrome:

A cookie associated with a cross-site resource at https://mysignalmasterserver/ was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5088147346030592 and https://www.chromestatus.com/feature/5633521622188032.

I've done some research and it appears that the cookie that socket.io uses by default is unnecessary and I am trying to disable it.

I see that I can just submit the "cookie: false" option when calling the listen method, but I can't figure out where this takes place in the Signal-Master realm of things.

I found a call to the listen method on the 6th line of sockets.js:
module.exports = function (server, config) {
    var io = socketIO.listen(server);

I tried altering it to:
module.exports = function (server, config) {
    var io = socketIO.listen(server, { cookie: false} );

But I still get the cookie warning in Chrome, so it appears that it was ineffective in halting the default cookie from being created.

I've tried several other similar alterations to various other Signal-Master, config, socket.io, and engine.io files with no luck at all. I feel like I'm just taking shots in the dark at this point and I felt I should finally ask for help.

TL;DR:  What file(s) should I alter in Signal-Master to set the "cookie" option to "false"?

**Comment by KevCoLaboratories** [comment]
**PER FIPPO:**

"

> var io = socketIO.listen(server, {cookie: false});
> in sockets.js seems like the correct solution. But I had some trouble verifying this works... because if you visit the page connecting to socket.io in Chrome, set cookiee to false, restart signalmaster and then reload the web page,
> Chrome still emits the warning even though the network tab in the devtools clearly shows there is no cookie. I suspect that is an oddity in Chrome because the cookie lifetime is bound to the session. Only restarting Chrome or visiting the site in a new private tab removed the console warning.
> 
> Maybe you ran into the same issue?

"

**Comment by KevCoLaboratories** [comment]
Outstanding!!

I was flushing the cache by refreshing the page with Ctrl-F5, but it didn't dawn on me to **close out of the incognito window completely to eliminate the session**, which now makes perfect sense.

I cannot thank you enough for your advice! It worked perfectly. **Issue: resolved.**

Truly, thank you. I was at my wit's end.
